.github/workflows/build_docker.yml

```diff
@@ -0,0 +1,43 @@
+name: Build Docker images
```

Comment:
@mscherer : The Docker build in the test is failing. Which may mean something or it may not. 
````
The command '/bin/sh -c npm run build' returned a non-zero code: 1
```

It looks like its being built with React? I know we previously were using NPM, setting the version to the latest of 12.22.x ..

Reply:
@mscherer : Maybe I should've seen you already noticed that.

Reply:
OK... I think I see what is happening.